### PR TITLE
feat(i18n): Add Brazilian Portuguese (#214)

### LINKED ISSUE #214: Add Language Support for Brazilian Portuguese
I have the branch ready to upload with support for Brazilian Portuguese, I just need permission to upload this

### PR DESCRIPTION
**Pull request by engrafaeldeon**
This pull request adds Brazilian Portuguese Translations.

Thank you for being able to contribute to this wonderful project.

Closes #214

### CODE REVIEW

File: .github/workflows/test.yml
Comment (fmaclen) [inline .github/workflows/test.yml:1]: Our existing CI workflow was miss-configured so I pushed this change to get the tests to run in this branch.
You can ignore it.

File: src/routes/settings/Interface.svelte [hunk 1]
```diff
@@ -31,7 +31,8 @@
 			{ value: 'en', label: 'English' },
 			{ value: 'es', label: 'Español' },
 			{ value: 'ja', label: '日本語' },
-			{ value: 'tr', label: 'Türkçe' }
+			{ value: 'tr', label: 'Türkçe' },
+			{ value: 'pt-br', label: 'Português do Brasil' }
```
Comment (fmaclen) [inline src/routes/settings/Interface.svelte:35]: Would you mind renaming the label `Português (Brasil)` so it's a little shorter?

File: tests/settings.test.ts [hunk 2]
```diff
@@ -232,4 +232,20 @@ test.describe('Locales', () => {
...
+
+			await page.evaluate(() => window.localStorage.clear());
+			await page.reload();
+			await expect(page.getByText('Server')).not.toBeVisible();
+			await expect(page.getByText('Sunucu')).toBeVisible();
+			expect(await page.evaluate(() => window.localStorage.getItem('hollama-settings'))).toContain(
+				'"userLanguage":"tr"'
+			);
+		});
+	});
```
Comment (fmaclen) [inline tests/settings.test.ts:250]: Looks like this is still testing Turkish.

**Review by fmaclen — CHANGES_REQUESTED** [review]
Thanks for submitting the PR 👍 

Most of the changes look good, I left you some minor comments below and the test needs to be updated for this new language.

Also, the linter check is [failing](https://github.com/fmaclen/hollama/actions/runs/11365631129). Running `npm run format && npm run lint` on your branch will likely fix the issue.

File: .github/workflows/test.yml
Comment (engrafaeldeon) [reply .github/workflows/test.yml:1]: thank you...

**Comment by engrafaeldeon** [comment]
I made the requested changes, should I close the PR and create and submit a new one?

**Review by fmaclen — APPROVED** [review]
Thanks for the contribution, this looks good!

No need to create a new PR, all of your changes are already here.
I'll get the PR merged for you.

**Pull request merged**